packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -328,7 +329,7 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ title, patientUuid, sh
               {showAddButton && (
                 <Button
                   kind="ghost"
-                  renderIcon={(props) => <Add size={16} {...props} />}
+                  renderIcon={() => <AddIcon size={16} />}
```

Comment:
Looks like there's a regression with the icon buttons:

![CleanShot 2024-07-08 at 10  27 05@2x](https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/f42f1797-66b2-4c82-bc8d-da7aed00005a)

Reply:
Aha! The props are important. I'll look into the color thing, but I think that's best fixed in the framework